fastapi_backend/api/index.py

```diff
@@ -0,0 +1 @@
+from app.main import app  # noqa: F401
```

Comment:
#noqa because ruff was complaining there's an unused import.

Reply:
This is the file from which vercell reads and creates the serveless functions.
It needs awsgi application: `app`